infracost-usage-example.yml

```diff
@@ -1177,37 +1178,25 @@ resource_usage:
   ibm_is_vpc.is_vpc:
     gigabyte_transmitted_outbounds: 5 # Monthly traffic leaving the VPC in GB
 
-  ibm_is_vpn_gateway.testVpcVpn:
+  ibm_is_vpn_gateway.test_vpc_vpn:
     monthly_connection_hours: 730 # Monthly number of connection hours
     monthly_instance_hours: 730 # Monthly number of instance hours
 
-  # 
-  # Power Systems Virtual Server
-  # 
-  ibm_pi_instance:
-    monthly_instance_hours: 730
-    storage: 20
-    cloud_storage_solution: 0
-    high_availability: 0
-    db2_web_query: 0
-    rational_dev_studio_licenses: 0
+  ibm_pi_instance.is_instance:
+    cloud_storage_solution: 0 # Cloud storage solution attachment, 0=false, 1=true, only valid for IBM i
```

Comment:
the trailing "i" in the desc should be removed

Reply:
I think that is the proper branding for the operating system